README.md

```diff
@@ -71,9 +71,9 @@ torchaudio.save("residual.wav", result.residual.cpu(), sample_rate)  # Everythin
 
 SAM-Audio supports three types of prompts:
 
-1. **Text Prompting**: Describe the sound you want to isolate using natural language
+1. **Text Prompting**: Describe the sound you want to isolate using natural language. To match training, please use lowercase and NP/VP format for text.
```

Comment:
I think a lot of people are probably not familiar with NP/VP (maybe you can provide some examples to illustrate)
```suggestion
1. **Text Prompting**: Describe the sound you want to isolate using natural language. To match training, please use lowercase and noun-phrase/verb-phrase (NP/VP) format for text (for example instead of "Thunder can be heard in the background" use "thunder").
```